frontend/public/components/utils/operator-backed-owner-references.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { OverviewItem } from '@console/shared';
+import { OwnerReferences } from './owner-references';
+
+export const OperatorBackedOwnerReferences: React.FC<OperatorBackedOwnerReferencesProps> = ({
```

Comment:
We prefer single default exports for component files.

Reply:
I've updated the component with default export. PTAL

Reply:
I've updated the component to export default OperatorBackedOwnerReferences.
PTAL